Scripts/passwordToggle.js

```diff
@@ -0,0 +1,23 @@
+const passToggleBtn = document.querySelectorAll('.passToggleBtn');
+
+
+// 비밀번호 숨김 구현
+function passwordToggle(event) {
+   if (event.target.classList.contains('notHidden')) {
+     //비밀번호를 숨김
+      event.target.classList.remove('notHidden');
+      event.target.previousElementSibling.setAttribute('type','password');
+
+   } else {
+      event.target.classList.add('notHidden');
+      event.target.previousElementSibling.setAttribute('type','text');
+   }
+};
+
+
+// 다중 토글 버튼 초기화 함수
+export default function toggleAssign() {
+   for (let btn of passToggleBtn) {
+      btn.addEventListener ('click', passwordToggle)
+   }
```

Comment:
for of loop에서 변수에 재할당해야 하는 경우가 아니면 const를 써주세요.

```suggestion
   for (const btn of passToggleBtn) {
      btn.addEventListener ('click', passwordToggle)
   }
```

Reply:
* mdn for...of 문서
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#description

Reply:
이게 왜 되죠; 당연히 안될거라 생각해서 let 선언한건데.... 알겠습니다